### PR TITLE
fix(blobber): fixed Used/Allocated

### LINKED ISSUE #1032: update getblobbers endpoint
from https://github.com/0chain/blobber/issues/127

[/v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/getblobbers](https://dev.0chain.net/miner01/v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/getblobbers) should be updated  in the storagesc code

- change `used` name to `allocated`
- add `used` for storage that is actually used by the client

### PR DESCRIPTION
**Pull request by cnlangzi**
## Fixes
-
## Changes
- renamed `Used` with `Allocation` on blobber node
- fixed `Used` for storage that is actually used by the client 
- 
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- 0chain: https://github.com/0chain/0chain/pull/1370
- blobber: https://github.com/0chain/blobber/pull/720
- gosdk: https://github.com/0chain/gosdk/pull/485
- system_test: https://github.com/0chain/system_test/pull/294
- zboxcli: https://github.com/0chain/zboxcli/pull/255
- zwalletcli: https://github.com/0chain/zwalletcli/pull/142
- zboxmobile: https://github.com/0chain/zboxmobile/pull/44

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/getblobbers  |
| gosdk  | fix/getblobbers  |
| zboxcli  | fix/getblobbers  |
| zwalletcli  | fix/getblobbers  |
| 0chain  | fix/getblobbers  |
| blobber  | fix/getblobbers  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/getblobbers  |
| gosdk  | fix/getblobbers  |
| zboxcli  | fix/getblobbers  |
| zwalletcli  | fix/getblobbers  |
| 0chain  | fix/getblobbers  |
| blobber  | fix/getblobbers  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: code/go/0chain.net/chaincore/chain/state.go [hunk 1]
```diff
@@ -205,6 +205,11 @@ func (c *Chain) updateState(ctx context.Context, b *block.Block, bState util.Mer
 	// check if the block's ClientState has root value
 	_, err = bState.GetNodeDB().GetNode(bState.GetRoot())
 	if err != nil {
+		logging.Logger.Error("update_state_failed",
```
Comment (peterlimg) [inline code/go/0chain.net/chaincore/chain/state.go]: The caller will log the error, let's not duplicate the logging

File: code/go/0chain.net/chaincore/chain/state.go [hunk 2]
```diff
@@ -573,6 +581,9 @@ func (c *Chain) mintAmount(sctx bcstate.StateContextI, toClient datastore.Key, a
 func (c *Chain) validateNonce(sctx bcstate.StateContextI, fromClient datastore.Key, txnNonce int64) error {
 	s, err := c.GetStateById(sctx.GetState(), fromClient)
 	if !isValid(err) {
+		logging.Logger.Error("validate nonce - error",
```
Comment (peterlimg) [inline code/go/0chain.net/chaincore/chain/state.go]: We don't need to log the error here, the only possible error here is 'node not found', which means the miner/sharder is not synced yet, and we will see this error in upper caller's logs.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/chaincore/chain/state.go (hunk 1 above)
Comment (cnlangzi) [reply code/go/0chain.net/chaincore/chain/state.go]: removed

File: code/go/0chain.net/chaincore/chain/state.go (hunk 2 above)
Comment (cnlangzi) [reply code/go/0chain.net/chaincore/chain/state.go]: removed

**Review by stewartie4 — APPROVED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**